### PR TITLE
Return token as purchaseToken

### PR DESCRIPTION
**Pull request by PEConn**

### CODE REVIEW

File: playbilling/src/test/java/com/google/androidbrowserhelper/playbilling/provider/MethodDataTest.java [hunk 1]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
```
Comment (andreban) [inline playbilling/src/test/java/com/google/androidbrowserhelper/playbilling/provider/MethodDataTest.java:1]: nit: 2021

```suggestion
// Copyright 2021 Google Inc. All Rights Reserved.
```

File: playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/provider/PaymentResult.java [hunk 2]
```diff
@@ -71,7 +71,9 @@ public void log() {
         @Override
         protected JSONObject toJson() throws JSONException {
             JSONObject obj = new JSONObject();
+            // "token" is deprecated, but kept around for backwatds compatibility.
```
Comment (andreban) [inline playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/provider/PaymentResult.java:74]: nit: typo.
```suggestion
            // "token" is deprecated, but kept around for backwards compatibility.
```

**Review by andreban — APPROVED** [review]
LGTM with couple of nits.

File: playbilling/src/test/java/com/google/androidbrowserhelper/playbilling/provider/MethodDataTest.java (hunk 1 above)
Comment (PEConn) [reply playbilling/src/test/java/com/google/androidbrowserhelper/playbilling/provider/MethodDataTest.java:1]: Technically the file is from 2020, I just forgot to add the copyright until 2021.

**Review by joycetoh8 — APPROVED** [review]
LGTM, thanks so much for the change Peter!

**Pull request merged**